mineinabyss-systems/src/main/kotlin/com/mineinabyss/pvp/adventure/AdventurePvpListener.kt

```diff
@@ -0,0 +1,86 @@
+package com.mineinabyss.pvp.adventure
+
+import com.derongan.minecraft.deeperworld.event.PlayerAscendEvent
+import com.derongan.minecraft.deeperworld.event.PlayerDescendEvent
+import com.derongan.minecraft.deeperworld.services.WorldManager
+import com.derongan.minecraft.deeperworld.world.section.inSectionOverlap
+import com.mineinabyss.components.playerData
+import com.mineinabyss.mineinabyss.core.MIAConfig
+import net.md_5.bungee.api.chat.ClickEvent
+import net.md_5.bungee.api.chat.TextComponent
+import org.bukkit.ChatColor
+import org.bukkit.entity.Player
+import org.bukkit.entity.Projectile
+import org.bukkit.event.EventHandler
+import org.bukkit.event.Listener
+import org.bukkit.event.entity.EntityDamageByEntityEvent
+import org.bukkit.event.player.PlayerMoveEvent
+
+class AdventurePvpListener : Listener {
+    @EventHandler
+    fun PlayerMoveEvent.onLeaveOrth() {
```

Comment:
Name your listeners by what they do, not when they get fired (ex you could call this one `promptPvpSelect`)

Reply:
I meant the actual function names not the class :p